CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Add constructors to layer classes so that types are emitted properly.
```

Comment:
nit. We technically don't "emit" any types from our JSDoc annotations. They just remain in the bundled `dist/index.js` as comments. Maybe "Refactor JSDoc comments to properly annotate Layer classes with constructor signatures."